manim/mobject/three_d/three_dimensions.py

```diff
@@ -109,11 +115,14 @@ def __init__(
         func: Callable[[float, float], np.ndarray],
         u_range: Sequence[float] = [0, 1],
         v_range: Sequence[float] = [0, 1],
```

Comment:
Based on the code which always expects them to have two floats, would it be possible for these two to be `tuple[float, float]`?

```suggestion
        u_range: tuple[float, float] = [0, 1],
        v_range: tuple[float, float] = [0, 1],
```

Reply:
Good idea